### ISSUE TITLE
Ability to prevent "uncheck" action on rows

### ISSUE THREAD

**Issue by staticdreams** [ag-grid/ag-grid#5431]
**I'm submitting a ...
```
[ ] bug report => see 'Providing a Reproducible Scenario'
[x] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[x] support request => see 'Requesting Community Support'
```

* **AG Grid version:** 28.1.0

Hello, I'm following up on [my previous feature request submission](https://github.com/ag-grid/ag-grid/issues/5385) from two weeks ago.

Thanks for implementing the feature that allows to show inactive checkboxes. I see it made its way to the newest release 28.1.0.

However, I'd like to reiterate on my question, as it is a similar feature, but not really what I wanted.
What I wanted to implement in my data table is a reversed functionality of what has been added to the latest release:

- Ability to pre-select (pre-check) certain rows on load without giving the possibility to uncheck them (imposing the selection on user, making it clear that the selected rows are mandatory)

Basically it is the same as the newly implemented ``isRowSelectable``, but something like ``isRowUnselectable``

From what I can see in the AG Grid pipeline, no such feature made it the backlog. Is there a way to request this feature?

P.S: we're a paying Enterprise Edition customer.

Thanks

**Comment by kiril-matev** [comment]
Hello @staticdreams!

As an AG Grid Enterprise customer you are entitled to use our support portal. Please contact our sales team at info@ag-grid.com and request an account.

Glad to hear you appreciate the new functionality we added in AG Grid 28.1.

You can implement the checkbox behavior you described as shown below using the checkboxSelection callback:
https://plnkr.co/edit/vmrCerpMGrne6fwZ?open=main.js

Please note that rows with year=2008 have disabled checkboxes which are checked and cannot be un-checked.

I hope this helps.

Thank you for using AG Grid!
